src/helpers/checkErrorMessage.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios'
+
+export default function (error: unknown) {
+  return error instanceof Error || axios.isAxiosError(error)
+    ? error.message
```

Comment:
<img width="698" alt="image" src="https://user-images.githubusercontent.com/7966131/191079419-d1e4fcb1-df04-4c79-82e2-b3fdd3ffbd2e.png">
It doesn't take the error text sent from the server

Reply:
Must be "Couldn't find a user with this username michel"

Reply:
Now under the text input the user will see an error message and in toast they will see a handled error from the package with more details